.github/workflows/kind.yaml

```diff
@@ -37,10 +37,15 @@ jobs:
           # Run test
           make integration-test
 
-          # Export logs
+      # collect logs always just in case the test fails
+      - name: Collect all logs
+        if: always()
```

Comment:
I'm not sure about this, but shouldn't expressions use the form `${{ <expression> }}`? In other words is it ok to use `always()` instead of `${{ always() }}` ?

Reply:
uh thanks! :)